src/content/introduction.md

```diff
@@ -40,8 +40,12 @@
 
 ---
 
+![Redux Flow](content/images/full-redux-flow.png "Redux Flow")
```

Comment:
I'm not sure this should go in the introduction. We can maybe even use this full circle diagram at the end, or in place of the other selector diagram I did...